src/content/self-hosted/install/releases.mdx

```diff
@@ -6,16 +6,52 @@ id: releases
 ---
 
 ## 1.8.0
-TBD
+10 May, 2023
 
 ### Features
+
```

Comment:
I'm missing also some mention to the new ingress controller that will be installed as part of Okteto in this version and how to keep using just one if customers wants

Reply:
this will be covered by @jpf-okteto PR